PWGHF/HFL/Tasks/taskSingleMuon.cxx

```diff
@@ -190,12 +192,14 @@ struct HfTaskSingleMuon {
       const auto charge(muon.sign());
       const auto chi2(muon.chi2MatchMCHMFT());
 
-      auto trkMFT = muon.template matchMFTTrack_as<MFTTracksExtra>();
-      if (reduceAmbMft && trkMFT.has_ambMftTrack()) {
-        continue;
-      }
-      if (reduceOrphMft && (!reduceAmbMft) && trkMFT.has_ambMftTrack()) {
-        continue;
+      if (muon.matchMFTTrackId() > 0) {
```

Comment:
Same as above
```suggestion
      if (muon.has_matchMFTTrack()) {
```

Reply:
Hi @fgrosa , thank you for the comments. The new syntax has been tested and uploaded.